share/spice/caniuse_lists/caniuse_lists.js

```diff
@@ -0,0 +1,164 @@
+
+        //pick only the required features based on category
+        for( var feature in data ) {
+
+            var obj = data[feature];
+            for(var search_term in result) {
+                if( obj['categories'].indexOf(result[search_term].toUpperCase()) != -1) {
+                    required_data.push(obj);
+                    break;
+                }
+            }
+        }
+
+        //sort by usage percentage, since there are many features
+        required_data.sort(function(a,b) {
+            return b.usage_perc_y - a.usage_perc_y;
+        });
+        
+        // Render the response
+        Spice.add({
+            id: 'caniuse_lists',
+            name: 'Can I Use',
+            data: required_data,
+            meta: {
+                sourceName: 'caniuse.com',
+                sourceUrl: 'http://caniuse.com/'
+            },
+            normalize: function(item) {
+                supported = {'ie': true,'firefox': true,'chrome': true,'safari': true};
+                return {
+                    title      : item.title,
+                    subtitle   : item.categories,
+                    description: item.description,
+                    ie         : getStatus('ie', item),
+                    chrome     : getStatus('chrome', item),
+                    firefox    : getStatus('firefox', item),
+                    safari     : getStatus('safari', item),
+                    ie_present : supported['ie'],
+                    chrome_present : supported['chrome'],
+                    firefox_present : supported['firefox'],
+                    safari_present : supported['safari'],
+                    ie_path    : DDG.get_asset_path('caniuse_lists', 'assets/ie-icon-16.png'),
+                    chr_path   : DDG.get_asset_path('caniuse_lists', 'assets/chrome-icon-16.png'),
+                    ff_path    : DDG.get_asset_path('caniuse_lists', 'assets/firefox-icon-16.png'),
+                    and_path   : DDG.get_asset_path('caniuse_lists', 'assets/android-browser-icon-16.png'),
+                    saf_path   : DDG.get_asset_path('caniuse_lists', 'assets/ios-safari-icon-16.png')
+                };
+            },
+            templates: {
+                group: 'text',
+                options: {
+                    footer: Spice.caniuse_lists.footer,
+                    moreAt: true
+                },
+                variants: {
+                    tileTitle: '2line-small',
+                    tileSnippet: 'large',
+                    tileFooter: '3line'
+                }
+            }
+        });
+
+        function getStatus(browser_name,item) {
+            var compatibility = item.stats[browser_name],
+                versions = Object.keys(compatibility),
+                index = 0,
+                minVersion = '--',
+                return_value,
+                support_type = {'yes': ['y'],'partial': ['a','a x','y x'], 'y': ['y #n'], 'a': ['a #n', 'a x #n']};
```

Comment:
can we break this out vertically? I would remove quoting on the keys as it's not necessary.